packages/react/src/jupyter/kernel/KernelExecutor.ts

```diff
@@ -258,11 +258,13 @@ export class KernelExecutor {
           break;
         case 'error':
           {
-            const { ename, evalue, traceback } = (
-              content as any as KernelMessage.IErrorMsg
-            ).content;
+            // NOTE: This block that was here previously cannot extract the properties consistently, 
+            //causing the Promise to never be rejected in case of an execution error.
+            // const { ename, evalue, traceback } = (
+            //   content as any as KernelMessage.IErrorMsg
+            // ).content;
```

Comment:
As discussed with @fcollonval the initial code shoudl be .

```
const { ename, evalue, traceback } = content as any as KernelMessage.IErrorMsg;
```

Could you try this way?

Reply:
@MarcosVn

Reply:
@eric, my tslint configuration says that traceback does not exist in `IErrorMsg`, should I ignore this?

![image](https://github.com/user-attachments/assets/2a851ed0-8ba2-49e3-bc37-9fd75fc1d179)

Here, content is `IReplyErrorContent` & `IExecuteCount`:

![image](https://github.com/user-attachments/assets/fbc0be0c-7c5e-4995-ad50-47b19835440a)

Reply:
I committed it as `const { ename, evalue, traceback } = content as any as KernelMessage.IReplyErrorContent`; - it worked here.

Reply:
this works for me in my vscode. What about you?

```
        case 'error':
          {
            const { ename, evalue, traceback } = content as KernelMessage.IReplyErrorContent & KernelMessage.IExecuteCount;
            this._executed.reject(
              `${ename}: ${evalue}\n${(traceback ?? []).join('\n')}`
            );
          }
```

Reply:
message crossing. I think you can remove the intermediary `as any`

Reply:
Updated, @echarles.

Reply:
:+1:  I can approve and merge unless you have more to do here. Maybe you can bump the version number to `0.16.1`  so I can release after the merge.

Reply:
I applied the version update here @echarles . The only point is that it was on `0.17.0`, and I set it to `0.17.1`.

Thanks in advance

Reply:
oops I bumped at the last merge... really sorry... can you revert back to 0.17.0 ?

Reply:
Reverted, @echarles.